docs/reST/ref/color.rst

```diff
@@ -214,6 +214,7 @@
       is useful if you want to unpack to r,g,b and not r,g,b,a. If you want to
```

Comment:
There should be a "DEPRECATED" bit in this documentation, shortly explaining why. 

For example:
![Capture](https://user-images.githubusercontent.com/46412508/182004931-c96f62cd-936a-48aa-8a98-630fb9cda874.PNG)

Although unlike this one, Color.set_length is not completely self explanatory, so you should keep the current documentation content as well.